up-l2/dispatchers/README.adoc

```diff
@@ -123,3 +123,17 @@ When a dispatcher is unable to dispatch an event for a given reason (queue full,
 .RPC Error Flow
 [#rpc-error-flow]
 image::rpc_flow.png[RPC Error Handling]
+
+== File Publish Handling
+
+When a uStreamer instance dispatcher is supporting file transfer, the file details will generally be passed by reference, and the transfer will be queued for delivery.  uP traffic priorities will affect when content can be delivered, and file size(s) along with available link bandwidth will also affect transfer duration.
```

Comment:
We don't pass file details by reference, we pass file metadata in UAttributes (that is what we are suppose to do).  The other text you wrote does not make sense (about priority etc...)